apps/chatbot/src/modules/vector_database.py

```diff
@@ -212,8 +229,8 @@ def create_documentation(
     # with open("full_text.txt", "w") as f:
     #     f.write(full_text)
 
-    logging.info(f"Number of documents with content: {len(documents)}")
-    logging.info(f"Number of empty pages in the documentation: {len(empty_pages)}. These are left out.")
+    logging.info(f"[vector_database.py] Number of documents with content: {len(documents)}")
```

Comment:
I think we can use the log formatter to print the name of the script and much more
more info [here](https://stackoverflow.com/questions/533048/how-to-log-source-file-name-and-line-number-in-python)

Reply:
You are right
(but now I remove the logs)